web/public/mgmt/config.yml

```diff
@@ -1233,19 +1233,21 @@ collections:
   - name: "funding-opportunities"
     label: "Fund Opps - Content"
     folder: "content/src/fundings"
-    summary: "/{{urlSlug}} - {{name}}"
+    summary: "[{{agency[0]}}] - {{name}}"
     identifier_field: "{{filename}}"
     slug: "{{filename}}"
     delete: true
+    sortable_fields: ["name", "agency"]
```

Comment:
dust - should we use displayName instead of name?

Reply:
I second that

Reply:
Thanks for catching this!